dotcom-rendering/cdk/lib/dotcom-rendering.ts

```diff
@@ -216,12 +233,19 @@ export class DotcomRendering extends GuStack {
 			vpcSubnets: { subnets: privateSubnets },
 			withoutImdsv2: true,
 		});
+		// ! Important !
+		// Ensure the ASG is attached to the load balancer
```

Comment:
Can you write a bit more on why this is important and perhaps provide a link to a doc?